divnoising/gaussianMixtureNoiseModel.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, **kwargs):
             self.max_signal=torch.Tensor(params['max_signal']).to(self.device)
             
             self.weight=torch.Tensor(params['trained_weight']).to(self.device)
-            self.min_sigma=np.asscalar(params['min_sigma'])
+            self.min_sigma=np.float(params['min_sigma'])
```

Comment:
The `np.float` should also raise a deprecation warning since numpy v1.20. This should do the trick:
```suggestion
            self.min_sigma=params['min_sigma'].item()
```

Reply:
Yes thanks ! Done !!